client/src/stores/initApp.js

```diff
@@ -56,6 +57,17 @@ const initApp = async (state) => {
           update.selectedConnectionId = selectedConnectionId;
         }
       }
+
+      const qsConnectionName = queryString.parse(window.location.search)[
+        'connectionName'
+      ];
+      if (qsConnectionName) {
```

Comment:
Could this also be updated to support `connectionId` as a parameter?

Reply:
done